packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -141,48 +169,74 @@ export default class AppStore {
     }
 
     registerLandingCompanyChangeReaction() {
-        const { client, common } = this.root_store.core;
+        const { client, common, ui } = this.root_store.core;
 
         this.disposeLandingCompanyChangeReaction = reaction(
             () => client.landing_company_shortcode,
             () => {
                 if (
-                    (!client.is_logged_in && client.is_eu_country) ||
-                    (client.is_eu && window.location.pathname === routes.bot) ||
+                    (client.is_eu &&
+                        window.location.pathname === routes.bot &&
+                        client.is_low_risk &&
```

Comment:
if the client is the `is_low_risk` then they can not be `is_eu`, so in this if condition always will pass false :thinking_face

Reply:
Removed `is_low_risk`

Reply:
@mahdiyeh-deriv removed `is_low_risk`